### ISSUE TITLE
false positive on 'split'

### ISSUE THREAD

**Issue by bingalls** [wapmorgan/PhpCodeFixer#10]
phpcf complains, to use preg_split() instead of deprecated split() even when it is not a function
You likely need to add code that checks for an opening ( parenthesis, to ensure a function.

"wapmorgan/php-code-fixer": "^2.0"
PHP 7.0.15 (cli)

```
<?php

class Banana {
  public $split = 'tasty';
}

$banana = new Banana();
$dessert['split'] = $banana->split;

print_r($dessert);
```

 PHP |             Type |                      File:Line | Issue
 5.3 | function         |                   cbfBug.php:8 | Function split is deprecated.

**Comment by wapmorgan** [comment]
Will be fixed this week

**Comment by wapmorgan** [comment]
Fixed in 2.0.3
